### PR TITLE
Improve python3 support

### PR DESCRIPTION
**Pull request by ybznek**
I have tried daemon with python3 on fedora23 and there were/are some import/... issues.

In this PL I will do the fixes.

It is NOT COMPLETE, and it has NOT been TESTED.

I hope, I will finish it today's evening.

### CODE REVIEW

File: bin/oscapd [hunk 1]
```diff
@@ -25,7 +25,10 @@ from openscap_daemon import version
 
 import os
 import logging
-import gobject
+try:
+    import gobject
+except ImportError:
+    from gi.repository import GObject as gobject
```
Comment (ybznek) [inline bin/oscapd]: I am quite not sure with this import. I will investigate it later.

**Comment by mpreisler** [comment]
@isimluk established a different way of checking whether we are in python3 or python2, check his contribution: https://github.com/OpenSCAP/openscap-daemon/pull/37/files

I don't care which way it's done but please let's do it consistently across the code-base.

**Comment by ybznek** [comment]
It has been tested with atomic scan on fedora23(python3) and fedora22(python2). I think you can merge it if you want.

Probably there are still some atomic-nonrelated python3 issues in daemon.

**Comment by mpreisler** [comment]
Thx

**Pull request merged**